### ISSUE TITLE
Connect "Raw transaction" transaction type action

### ISSUE THREAD

**Issue by ArmandoGraterol** [JoinColony/colonyDapp#3721]
### Description

This issue corresponds to the `Raw transaction` transaction type inside the "Control safe" dialog. In order to make the transaction, we are going to use a zodiac bridge to execute the transactions across chains.

The transaction is expected to work for cross-chain safes/contracts.

## Implementation

* Steps to get the dev env ready for testing transactions:
```
0. Have your normal dev env already running.
1. cd src/lib/colonyNetwork 
2. git fetch
3. git checkout maint/cross-chain-safe 
4. git submodule update --init
5. yarn
6. yarn provision:safe:contracts
7. cd lib/safe-contracts then yarn
8. cd ../.. (back into colonyNetwork)
9. bash scripts/start-bridging-environment.sh
10. node scripts/start-bridging-environment.js
```
You'll see this in the console if the steps were run successfully:
```
Bridge Monitor running
Home bridge address: 0xE0EeC1f1d5Cc344206710Ad526E180e5CeEE5B1c
Foreign bridge address: 0x312787b78BA0A2C4e9092a7871E49c421c24e7Fa
Gnosis Safe address: 0xC33533885f666baB63243b9E0B8432602d8dDb52
Zodiac Bridge module address: 0x0a72DCB0584A80cada11cC95aB2386f23225C63d
```

### This issue is complete when...
- [ ] Submitting the form executes a transaction/many transactions (in the case of a multi transaction).

### Suggestions

* These links could be helpful for testing/development:
Safes for testing: https://www.gnosis-safe.io/app/rinkeby
Safes for testing: https://dev.gnosis-safe.io/
How to find the emitted events if needed: https://docs.ethers.io/v4/api-contract.html?highlight=events#filtering-events
Example of how to make the transactions (In the context of Network, however): https://github.com/JoinColony/colonyNetwork/commit/e4484246889a4d24b19e58e9a496330e00a77f0f

**Comment by ArmandoGraterol** [comment]
Hey @willm30! I'll be taking a closer look at this as setting this up is part of #3754. Olga suggested having the setup of the libraries as a separate issue, seeing that every transaction is going to need to do it so we just do it in 1 place to avoid repeating code but the thing about Ethers v5 that you mentioned might be it

**Comment by willm30** [comment]
> Hey @willm30! I'll be taking a closer look at this as setting this up is part of #3754. Olga suggested having the setup of the libraries as a separate issue, seeing that every transaction is going to need to do it so we just do it in 1 place to avoid repeating code but the thing about Ethers v5 that you mentioned might be it

Yes good idea. I'll wait until that one's complete 👍

**Comment by ArmandoGraterol** [comment]
@willm30 Hey Will! You may have noticed already but #3758 was merged yesterday, so the initial setup should be done and this issue ready to be implemented

**Comment by willm30** [comment]
MERGED #3765
